tests/model_hub_tests/tf_tests/requirements.txt

```diff
@@ -5,3 +5,5 @@ pytest-html
 tensorflow
```

Comment:
Also, I am thinking to rename it again. Really sorry or this. Now we have `tests` in both names `model_hub_tests` and `tf_tests`. To have it more ideal, let us rename `tf_tests` -> `tensorflow`. This way we will know location of each FW models validation.

Reply:
No problem. Done.